core-web/libs/template-builder/src/lib/template-builder.module.ts

```diff
@@ -33,7 +34,7 @@ import { TemplateBuilderComponent } from './components/template-builder/template
         DividerModule
     ],
     declarations: [TemplateBuilderComponent],
-    providers: [DotTemplateBuilderStore],
+    providers: [DotContainersService, DotTemplateBuilderStore],
```

Comment:
Do we need to import the `DotContainersService` in order to use the containers directive?